libs/features/pages/booklets-page/src/lib/components/booklets-hero-section/booklets-hero-section.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'elewa-website-booklets-hero-section',
+  templateUrl: './booklets-hero-section.component.html',
+  styleUrls: ['./booklets-hero-section.component.scss'],
+})
+export class BookletsHeroSectionComponent {
+  textData = {
+    title: 'Booklets',
+    subtitle: '',
+    description: 'Your organisation has the potential to turn every task into an opportunity to learn. From growing crops, to nursing someone to good health, to developing the next-generation technology, let’s work together to make the magic happen.'
+  };
+  heroButtonData = {
+    text: 'Unlock your potential'
+  }
+}
```

Comment:
To keep the component lean and ensure readability and maintainability, the 'textData' and 'heroButtonData' variable content can be moved to a Typescript file within the data module and referenced accordingly.